problems/multiplesOfThreeOrFive.js

```diff
@@ -24,8 +24,15 @@
 * Notice that 15 is a multiple of 3 and 5 BUT it is only added ONCE.
 */
 
-function multiplesOfThreeOrFive() {
-
-}
+const multiplesOfThreeOrFive = (n) => {
+    let sum = 0;
+      for(let i = 0; i <= n; i++) {
+            if(el % 3 === 0 || el % 5 === 0){
+                sum += (el);
+                return sum;
+         }
+       
+        });
```

Comment:
This test failed because of the `)` in line 35, thats why its important to format your code